### ISSUE TITLE
up env crashes 

### ISSUE THREAD

**Issue by cloudkite** [apex/up#427]
1. Add env without first specifying the non-stage specific env
`up env add -s development HELLO "world"`

2. list env
`up env ls`

```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/tj/simpletable.(*Table).transposeCells(0xc420106080, 0xc42049f888, 0x0, 0x0, 0xc4203f0050, 0x2d0a618, 0x0)
        /Users/tj/dev/src/github.com/tj/simpletable/table.go:264 +0x2d5
github.com/tj/simpletable.(*Table).prepareColumns(0xc420106080)
        /Users/tj/dev/src/github.com/tj/simpletable/table.go:245 +0xa7d
github.com/tj/simpletable.(*Table).String(0xc420106080, 0x0, 0x1)
        /Users/tj/dev/src/github.com/tj/simpletable/table.go:34 +0x166
github.com/tj/simpletable.(*Table).Println(0xc420106080)
        /Users/tj/dev/src/github.com/tj/simpletable/table.go:65 +0x2f
github.com/apex/up/internal/cli/env.list.func1(0xc4201db830, 0x1010a33, 0x19f6be0)
        /Users/tj/dev/src/github.com/apex/up/internal/cli/env/env.go:92 +0x77f
github.com/tj/kingpin.(*actionMixin).applyActions(0xc420225698, 0xc4201db830, 0x0, 0x0)
        /Users/tj/dev/src/github.com/tj/kingpin/actions.go:33 +0x6d
github.com/tj/kingpin.(*Application).applyActions(0xc420089d40, 0xc4201db830, 0x0, 0x0)
        /Users/tj/dev/src/github.com/tj/kingpin/app.go:570 +0xe1
github.com/tj/kingpin.(*Application).execute(0xc420089d40, 0xc4201db830, 0xc420226e40, 0x2, 0x2, 0x0, 0x0, 0x1, 0xc42000e5a8)
        /Users/tj/dev/src/github.com/tj/kingpin/app.go:406 +0x8f
github.com/tj/kingpin.(*Application).Parse(0xc420089d40, 0xc42000c070, 0x1, 0x1, 0x274a48a78000, 0x1, 0xc4202226c0, 0xc420229ed0)
        /Users/tj/dev/src/github.com/tj/kingpin/app.go:222 +0x1f2
github.com/apex/up/internal/cli/app.Run(0x1c057b0, 0x9, 0x0, 0x0)
        /Users/tj/dev/src/github.com/apex/up/internal/cli/app/app.go:16 +0xed
main.run(0x1b55699, 0xa)
        /Users/tj/dev/src/github.com/apex/up/cmd/up/main.go:69 +0x206
main.main()
        /Users/tj/dev/src/github.com/apex/up/cmd/up/main.go:45 +0x8a
```

**Comment by tj** [comment]
yikes I'll take a look right away

**Comment by tj** [comment]
oh I think it has something to do with development being explicitly passed, I had been just using `up env add FOO BAR` for dev (but acts as a default for others too). I'll get that fixed

**Comment by tj** [comment]
ah yeah my bad `for _, name := range []string{"all", "staging", "production"} {`

**Comment by cloudkite** [comment]
@tj My usecase was for some env variables I want to set them explicitly for each stage so that it doesn't fallback to some unexpected value.  Will this be supported? I'm hoping that `up env -s development add FOO BAR` will not set the fallback/default value. Thanks 👍

**Comment by tj** [comment]
Fixed in `0.1.5`, `up upgrade` to grab that :D sorry about that! Dumb little oversight, but yep it'll behave like you mention without a fallback if you always specify the `--stage`

**Comment by cloudkite** [comment]
Awesome thanks for the quick fix ❤️
